src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -8396,6 +8400,10 @@ ExpressionSyntax node8
             Assert.Equal(MethodKind.BuiltinOperator, symbol1.MethodKind);
             Assert.True(symbol1.IsImplicitlyDeclared);
 
+            var synthesizedMethod = compilation.CreateBuiltinOperator(
```

Comment:
>CreateBuiltinOperator [](http://example.com/codeflow?start=48&length=21)

Similarly, we should consider adding similar validation to Microsoft.CodeAnalysis.Test.Utilities.TestOperationVisitor.VisitBinaryOperator

Reply:
done.